esm-samples/jsapi-angular-cli/README.md

```diff
@@ -2,50 +2,45 @@
-
-*angular.json*
-
-```json
-  "allowedCommonJsDependencies": [
-    "moment"
-  ],
-```
----
-
 ## Get Started
 
-Run `npm install` and then start adding modules.
+Run `npm install` and then start adding functionality.
 
 For a list of all available `npm` commands see `scripts` in `package.json`.
 
-For additional information, see the [Build with ES modules](https://developers.arcgis.com/javascript/latest/es-modules/) Guide topic in the SDK.
+## Known issues
 
-## TypeScript
+* The Maps SDK compatibility with Webpack versions prior to `5.84.0` was deprecated at 4.27 and will be removed at 4.29 (Q1 2024).
 
-Currently, due to limitations in TypeScript, the APIs [autocasting](https://developers.arcgis.com/javascript/latest/programming-patterns/#autocasting) functionality works best in non-TypeScript applications. No changes are required if you are already using the API without any TypeScript build errors.
+* When using Angular 17, for better application load performance with distribution builds it is recommended to use the [Webpack builder](https://angular.io/guide/esbuild#using-the-application-builder) instead of the default esbuild configuration. This can be configured in the project's `angular.json` file. At this time esbuild does not offer the ability to customize chunking. That may result in the creation of a significant number of small on-disk bundled files, as well as an increase in the number of files requested by the application at startup. Here is an example snippet:
 
-This sample implements strict class initalization, for more information visit the [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization) page.
+*angular.json - Webpack configuration*
 
-Check the online Angular documentation for their minimum TypeScript requirements for each release.
+```json
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/",
+            "index": "src/index.html",
+            "main": "src/main.ts",
+            "polyfills": [
+              "zone.js"
+            ],
+``` 
 
-### Development server
+* Some of the SDKs widgets may not work correctly in Angular 16+. This is due to a Calcite bug affecting the enabling/disabling buttons via two-way data binding: https://github.com/Esri/calcite-design-system/issues/7729. The bug applies to all Calcite versions greater than `1.2.0`. A temporary workaround is to disable zone.js monkey patching of DOM `click` events by setting this flag: `windows.__zone_symbol__UNPATCHED_EVENTS = ['click']`. More information is available in [Issue #481](https://github.com/Esri/jsapi-resources/issues/481#issuecomment-1687048980).
```

Comment:
First link in this paragraph should be a markdown link. Looks like the period in the end is included and link breaks.